# HW 2



### Step 1: Load the python libraries you will need for this project 

In [1]:
#imports
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np

%matplotlib inline

### Step 2: Read in your data set 

In [2]:
#Read in data from source 
url = "https://raw.githubusercontent.com/ga-students/DS-SF-24/master/Data/admissions.csv"
df_raw = pd.read_csv(url)
df_raw.head(5)

admit    gre   gpa  prestige
0      0  380.0  3.61       3.0
1      1  660.0  3.67       3.0
2      1  800.0  4.00       1.0
3      1  640.0  3.19       4.0
4      0  520.0  2.93       4.0

## Questions
#### Question 1. How many observations are in our dataset? 


In [6]:
len(df_raw)

400

Answer: 400

#### Question 2. Create a summary table

In [7]:
df_raw.describe()

admit         gre        gpa    prestige
count  400.000000  398.000000  398.00000  399.000000
mean     0.317500  588.040201    3.39093    2.486216
std      0.466087  115.628513    0.38063    0.945333
min      0.000000  220.000000    2.26000    1.000000
25%      0.000000  520.000000    3.13000    2.000000
50%      0.000000  580.000000    3.39500    2.000000
75%      1.000000  660.000000    3.67000    3.000000
max      1.000000  800.000000    4.00000    4.000000

#### Question 3. Why would GRE have a larger STD than GPA?

Answer: they are on different scales. gre_min = 220 gre_max = 800 and gpa_min = 2.26 gpa_max = 4.00

#### Question 4. Drop data points with missing data


#### Question 5. Confirm that you dropped the correct data. How can you tell? 

#### Question 6. Create box plots for GRE and GPA

In [ ]:
#boxplot 1 GRE


In [ ]:
#boxplot 2 GPA



#### Question 7. What do these plots show?

Answer: 

#### Question 8.  If our model had an assumption of a symmetric distribution would we meet that requirement? 

Answer: 

#### Question 9: Add dummy variables for GRE scores. Top 10% are considered very high. next 25% of GRE scores are considered high, next 50% are considered medium and lowest 15% are considered Low. 

#### By a scatterplot explore the association between GRE based on GPA. For each observation, color according to perstige. 